dotnet/src/webdriver/Command.cs

```diff
@@ -123,7 +124,12 @@ public override string ToString()
         /// <returns>A <see cref="Dictionary{K, V}"/> with a string keys, and an object value. </returns>
         private static Dictionary<string, object> ConvertParametersFromJson(string value)
         {
-            Dictionary<string, object> parameters = JsonConvert.DeserializeObject<Dictionary<string, object>>(value, new ResponseValueJsonConverter());
+            var jsonSerializerOptions = new JsonSerializerOptions
+            {
+                Converters = { new ResponseValueJsonConverter() }
+            };
```

Comment:
In terms of performance optimization, I would extract `jsonSerializerOptions` in places like this one to `private static readonly` fields.

Reply:
Fixed in `Command` and `Response` classes as most affected places.